src/privileged/FirefoxMonitor.jsm

```diff
@@ -194,11 +194,32 @@ this.FirefoxMonitor = {
   },
 
   _loadBreachesTimer: null,
+  _breachesLastModified: 0,
   async loadBreaches() {
-    // TODO: check first if the list of breaches was updated
-    //       since we last checked, before downloading it.
-    //       (pending repsonse from Troy about how to do this)
-    let response = await fetch(this.breachListURL);
+    let response;
+    try {
+      response = await fetch(this.breachListURL, {
```

Comment:
Ah, I didn't notice this last time, but this probably needs `credentials: omit` (https://developer.mozilla.org/en-US/docs/Web/API/Request/credentials) to avoid sending cookies since we're in chrome-privilege land.

Reply:
Good call, done.